lua/nvim-treesitter/endwise.lua

```diff
@@ -76,11 +76,22 @@ local function endwise(bufnr)
     col = col - 1
 
     local root_lang_tree = parsers.get_parser(bufnr)
-    local root = ts_utils.get_root_for_position(row, col, root_lang_tree)
+    local tree_at_pos = root_lang_tree:language_for_range({row, col, row, col})
+    lang = tree_at_pos:lang();
```

Comment:
I'll merge then remove the semi-colon in a separate commit since it'll be quicker.